paper/paper.Rmd

```diff
@@ -24,140 +25,735 @@ knitr::opts_chunk$set(echo = TRUE)
 
 # Introduction
 
```

Comment:
I reworked the intro slightly to smooth the jumps between concepts a bit. Let me know what you think!